javascript/builtins/RegExp.json

```
@@ -426,6 +426,57 @@
             }
           }
         },
+        "hasIndices": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/RegExp/hasIndices",
+            "support": {
+              "chrome": {
+                "version_added": "90"
+              },
+              "chrome_android": {
+                "version_added": "90"
+              },
+              "edge": {
+                "version_added": "90"
+              },
+              "firefox": {
+                "version_added": "88"
+              },
+              "firefox_android": {
+                "version_added": "88"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              },
+              "samsunginternet_android": {
+                "version_added": false
+              },
+              "webview_android": {
+                "version_added": "90"
+              }
+            },
+            "status": {
+              "experimental": true,
```

Sorry, I ought to have caught this before. Generally, we set `experimental` to `false`, when there's two or more implementations.

```suggestion
              "experimental": false,
```

---

Thank you for the note! Changed it to `false`.